csharp/autobuilder/Semmle.Autobuild/BuildActions.cs

```diff
@@ -141,12 +141,8 @@ ProcessStartInfo GetProcessStartInfo(string exe, string arguments, string workin
                 pi.WorkingDirectory = workingDirectory;
 
             // Environment variables can only be used when not redirecting stdout
-            if (!redirectStandardOutput)
-            {
-                pi.Environment["UseSharedCompilation"] = "false";
```

Comment:
I think we still need this environment variable in the case that we have a build script on Linux on dotnet core <=2.2. In this case, I see no other mechanism in which to disable shared compilation, and if we don't then we crashed the build. So, I think we also need a mechanism/parameter to detect the .NET Core that covers this environment variable.